design/src/App.scss

```diff
@@ -188,3 +188,103 @@ $hover-color: #f8cc80;
         }
     }
 }
+
+
+
+// NAVBAR
+
+.navbar {
+  height: 96px;
+  background: #25271C;
+  position: sticky;
+
+  .navbar-content {
+    height: auto;
+    padding: 30px;
+    margin: 0 auto;
+    padding-top: 20px;
+
+    .navbar-logo {
+        float:left;
+    }
+    
+    .navbar-menu {
+        float:right;
+        position: relative;
+        top: 20px;
+    }
+  }
+}
+
+.menu-links {
+    list-style: none;
+
+    li {
+        display: inline-block;
+        margin-left: 36px;
+        font-family: Archivo;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 20px;
+        line-height: 22px;
+        text-align: right;
+        text-transform: uppercase;
+        color: #fff;
+        cursor: pointer;
+        border-bottom: 2px solid transparent;
+        padding-bottom: 5px;
+    }
+
+    li:hover {
+        color: #f8cc80;
+        border-bottom: 2px solid #f8cc80;
+    }
+
+    li:active {
+        color: #fff;
+        border-bottom: 2px solid #fff;
+    }
+}
+
+// menu burger
+.menu-burger {
+    position: relative;
+    width: 55px;
+    height: 4px;
+    background: rgb(255, 255, 255);
+    transition: all .1s ease-in-out;
+    margin: 0 auto;
+    top: 5px;
+}
+
+.menu-burger::before,
```

Comment:
Dessa skulle kunna flyttas in under `menu-burger`- selektorn eftersom att ni kör Sass :)